openfisca_us/parameters/tax/ald/misc/alimony/paid.yaml

```
@@ -0,0 +1,7 @@
+description: Adjustment for alimony-paid haircut
+metadata:
+  unit: currency-USD
```

This and other haircuts are proportions
```suggestion
  unit: /1
```

---

Fixed